pkg/kubectl/cmd/util/cmdproto/cmdproto.go

```diff
@@ -0,0 +1,88 @@
+package cmdproto
+
+import (
```

Comment:
how about moving this to: k8s.io/kubectl/pkd/framework/proto?

Reply:
Could we do that after kubectl fully segregated from main repo. I think there's a circular dependency problem currently.

Reply:
We should break those dependencies by defining interfaces and using those.